indra/llprimitive/llgltfmaterial.h

```diff
@@ -49,6 +49,11 @@ class LLGLTFMaterial : public LLRefCount
     // default material for reference
     static const LLGLTFMaterial sDefault;
 
+    static const char* ASSET_VERSION;
+    static const char* ASSET_TYPE;
```

Comment:
these should possibly be `static const char* const` (realizing belatedly that I probably should have noticed this on the server side review already)